packages/browser/src/storage.ts

```diff
@@ -259,6 +260,7 @@ export const localStore: PersistentStore = {
 // This solves issues with cookies having too much data in them causing headers too large
 // Also makes sure we don't have to send a ton of data to the server
 const COOKIE_PERSISTED_PROPERTIES = [
+    DEVICE_ID,
```

Comment:
agree with the robot, i don't know what tests there are already but we should have something on this and assert device id is included  
  
am actually amazed that it didn't cause a failing test here - i guess they all mock posthog or have inmemory persistence so it would show actually...